packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/ModelGrounding.java

```diff
@@ -0,0 +1,14 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.modelparts;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import java.util.Map;
+
+@Data
+@Accessors(chain = true)
+public class ModelGrounding {
+	private Map<String, Object> identifiers;
```

Comment:
Do we know what these objects are?

Reply:
Will ask the ta-4 channel. I dont think so. https://github.com/DARPA-ASKEM/Model-Representations/blob/main/petrinet/petrinet_schema.json#LL144C1-L158C5